packages/tentales/src/service/create-service-callers.ts

```diff
@@ -9,7 +11,18 @@ function getServiceConfig(serviceName: string, config: any): ServiceConfig {
 }
 
 export function createServiceCallers(config: Config): ServiceCaller[] {
-  const bearer = `Bearer ${mintToken(config.auth.serverSecret)}`
+  const headers = new Headers()
+  headers.append("Accept", "application/json")
+  headers.append("Content-Type", "application/json")
+
+  const secret = path(["auth", "serverSecret"], config) as
+    | string
+    | Buffer
+    | undefined
+
+  if (secret !== undefined) {
```

Comment:
R.isNil()

Reply:
I this style cause typescript was able to figure out that I manually checked the `undefined`-case, but it should probably be able to do the same with `if (secret)` - will try it!